### PR TITLE
LPS-194217 | Cannot change favicon from favicon extension | master | reapply LPS-189853 after revert

### PR DESCRIPTION
**Pull request by epark18**
https://liferay.atlassian.net/browse/LPS-194217
https://liferay.atlassian.net/browse/LPS-189853

resend of https://github.com/liferay-lima/liferay-portal/pull/4616

Hi @liferay-lima, I have reapplyed the earlier reverts to LPS-189853 and added the changes made in LPS-194217.

### CODE REVIEW

**Comment by liferay-lima** [comment]
ci:test:sf

**Comment by liferay-lima** [comment]
ci:test:relevant

**Comment by epark18** [comment]
ci:test:sf

**Comment by Tim-Cao** [comment]
ci:test:echo

**Comment by Tim-Cao** [comment]
ci:test:lima

**Comment by epark18** [comment]
ci:test:relevant

**Comment by epark18** [comment]
ci:test:echo

**Comment by epark18** [comment]
ci:test:lima

**Review by ambrinchaudhary — APPROVED** [review]

**Comment by ambrinchaudhary** [comment]
If tests passes we can forward this.

**Comment by Jeremy-Ch** [comment]
Hi @BarbaraCabrera Some issues are still related to this pr. Could you help to take a look? Thanks.

**Case1: Cannot change the display style in item selector**

1. Go to People> Memberships
2. Click “+“
3. Change the display style

**Case2: Cannot change the site scope in the item selector**

1. Go to add a blog entry
2. Select a related assets
3. Click the “Sites and Libraries“ breadcrumb in the item selector

The above issues all have below errors in the browser console:
image

/cc @Tim-Cao

**Comment by Tim-Cao** [comment]
Hi @BarbaraCabrera, this issue still happens on this PR.
Would you mind checking?
https://liferay.atlassian.net/browse/LPS-194231

**Comment by BarbaraCabrera** [comment]
Hey, guys!

@Tim-Cao I have tested the issue and it is true that it still happens in these cases, however it is due to an overflow problem with Eric's branch. That is, as the branch is not overflowed with master, when I try to test it, it still gives these errors. That can be easily seen because if I test the branch that I sent (link) we see that it is corrected and if I try with the commits in a new branch rebased with master they work. So this particular problem would be solved with a rebase.
I leave a video attached that it works correctly.

However, for the problem of (@Jeremy-Ch) if you need to make a change, and is that instead of making a `searchContainerOnHandler.dispose();` we need to make a `searchContainerOnHandler.detach();`

So if @ambrinchaudhary agrees with this change, I'll upload a pr with that change and the master rebase and fix both problems, since I can't edit Eric's branch. 

https://github.com/liferay-lima/liferay-portal/assets/93203423/50865ec7-230b-460f-bc0a-3c05280d4726

**Comment by ambrinchaudhary** [comment]
> However, for the problem of (@Jeremy-Ch) we do need to make one change, and is that instead of making a searchContainerOnHandler.dispose(); we need to make a searchContainerOnHandler.detach();

Sure! Go ahead and change it ;-). Thanks @BarbaraCabrera

**Comment by BarbaraCabrera** [comment]
ci:close
